### PR TITLE
[SourceKitten] update to 0.4.4

### PR DESCRIPTION
**Pull request by jpsim**

### CODE REVIEW

File: lib/jazzy/SourceKitten/Rakefile [hunk 1]
```diff
@@ -1,18 +0,0 @@
-# This Rakefile just ensures that the necessary framework symlinks are in place
-# when installed as a gem, as RubyGems doesn't play nice with symlinks.
-
-task :prepare do
```
Comment (segiddins) [inline lib/jazzy/SourceKitten/Rakefile]: you sure we can get rid of this?

**Comment by segiddins** [comment]
@jpsim can you summarize what the vending changes are, please?

**Comment by jpsim** [comment]
> @jpsim can you summarize what the vending changes are, please?

I had mistakenly removed `lib/jazzy/SourceKitten/Rakefile` but have now added it back in with a few modifications to make it glob recursively. I've confirmed this installs properly via `bundle exec rake install` and using it to build Realm Swift docs.

The changes I made to SourceKitten's runtime search paths make it so it no longer searches for frameworks in an absolute directory like it used to (`/Library/Frameworks/`). This caused problems when installing it directly (via `make install`) vs through jazzy or Homebrew.

**Comment by segiddins** [comment]
:+1:
